src/uct/ib/rc/verbs/rc_verbs_common.c

```diff
@@ -189,3 +189,11 @@ ucs_status_t uct_rc_verbs_wc_to_ucs_status(enum ibv_wc_status status)
         return UCS_ERR_IO_ERROR;
     }
 }
+
+void uct_rc_verbs_common_packet_dump(uct_base_iface_t *iface, uct_am_trace_type_t type,
```

Comment:
minor: may be move it to header & mark as inline?

Reply:
slow path function - i'd say it's not needed